gammapy/spectrum/core.py

```diff
@@ -374,6 +386,35 @@ def excess_vector(self):
         """
         return self.on_vector + self.off_vector * self.alpha * -1
 
+    def peek(self, figsize=(15, 5)):
+        """Quick-look summary plots."""
+        import matplotlib.pyplot as plt
+        plt.style.use('ggplot') 
+
+        fig, ((ax1, ax2), (ax3, ax4)) = plt.subplots(2,2,figsize=figsize)
+        self.on_vector.plot(ax=ax1, label='Total counts', histtype='step',
+                            color='darkred', alpha=0.8, lw=2)
+        self.off_vector.plot(ax=ax1, label='Background estimate',
+                             histtype='step', color='darkblue',
+                             alpha=0.8, lw=2)
+        ax1.legend(numpoints=1)
+        ax1.set_title('Counts')
+        # TODO: Replace with total spectrum stats
+        ax2.text(0.2, 0.9, 'Exposure: {}'.format(self.exposure))
+        ax2.text(0.2, 0.85, 'Alpha: {}'.format(self.alpha))
+        ax2.axis('off')
+        ax2.set_title('Spectrum Stats')
+        self.aeff.plot(ax=ax3)
+        ax3.set_title('Effective Area')
+        self.edisp.plot_matrix(ax=ax4)
+        ax4.set_title('Energy Dispersion')
+
+        import seaborn as sns
+        sns.despine(fig) 
+
+        plt.show()
```

Comment:
I think calling `show` here means that the function can't be tested. It will hang when you run it via pytest.
For now, please add a test that executes this peek method once and make sure it runs through.

We can play around with `show` and try to find a convenient solution and consistently apply it to all `peek` or even `plot` functions. But again ... better to figure this out once and do it in a separate PR.